### DISCUSSION TITLE
Using Pulsar CLI tools

### DISCUSSION THREAD

**Discussion by g41797** [apache/pulsar#22780] [Q&A] [answered]
According to docs:
  "All Pulsar command-line tools can be run from the bin directory of your [installed Pulsar package](https://pulsar.apache.org/docs/next/getting-started-standalone/)"

I am going to develop client application for Pulsar.

But Pulsar will be within Docker container.

During development I'd like to use Pulsar CLI tools
without installation of Pulsar package.

Is it possible?

Thanks

**Comment by lhotari** [comment, marked as answer]
> During development I'd like to use Pulsar CLI tools without installation of Pulsar package.
> 
> Is it possible?

There's no separate distribution for the Pulsar CLI tools alone. There's a [distribution for Pulsar Shell](https://pulsar.apache.org/download/#shell). Pulsar Shell is also a CLI tool. Please take a look at the [Pulsar Shell documentation](https://pulsar.apache.org/docs/3.2.x/administration-pulsar-shell/) for the installation and usage.
